### ISSUE TITLE
v0.12: TF_VAR_ environment not being used to override default values

### ISSUE THREAD

**Issue by alexng-canuck** [hashicorp/terraform-plugin-sdk#153]
### Terraform Version

```
github.com/hashicorp/terraform v0.12.0-alpha4.0.20190429183344-30199600eb15
```

### Terraform Configuration Files

```hcl
variable "identity_provider_metadata_file" { default = "sampleFederationMetadata.xml" }

resource "oci_identity_identity_provider" "test_identity_provider" {
  #Required
  compartment_id = "${var.tenancy_ocid}"
  description    = "${var.identity_provider_description}"
  metadata       = "${file("${var.identity_provider_metadata_file}")}"
...
}
```

Along with this configuration, we set an environment variable called `TF_VAR_identity_provider_metadata_file` and set it to `/usr/blahblahblah/sampleFederationMetadata.xml`.
### Expected Behavior

In v0.11, the `TF_VAR_` prefixed environment variable would have overridden the default value of the configuration.

### Actual Behavior

In v0.12, the default value is still being used, resulting in errors when passing an invalid file path to the `file()`

```
    testing.go:568: Step 0 error: errors during plan:
        
        Error: Error in function call
        
          on /var/folders/6r/8fk5dmbj4_z3sl0mc_y_fhjw0000gn/T/tf-test209101720/main.tf line 45:
          (source code not available)
            |----------------
            | var.identity_provider_metadata_file is "sampleFederationMetadata.xml"
        
        Call to function "file" failed: no file exists at
        sampleFederationMetadata.xml.
```

### Steps to Reproduce

`terraform apply`

### Additional Context

This was caught via a test case that had previously worked with v0.11 plugin SDKs. In v0.12, the same test fails.

**Comment by hazcod** [comment]
Wait for `0.12.11` I suppose?

**Comment by kmoe** [comment]
This is fixed in #262, the new binary acceptance test driver, which can be enabled from v1.7.0 onwards. Environment variables in the acceptance test environment are now passed transparently to the `terraform` process used to run the tests.
